ql/src/test/org/apache/hadoop/hive/ql/io/orc/TestOrcFile.java

```diff
@@ -325,7 +326,7 @@ public void testReadFormat_0_11() throws Exception {
         + "binary,string1:string,middle:struct<list:array<struct<int1:int,"
         + "string1:string>>>,list:array<struct<int1:int,string1:string>>,"
         + "map:map<string,struct<int1:int,string1:string>>,ts:timestamp,"
-        + "decimal1:decimal(38,18)>", readerInspector.getTypeName());
+        + "decimal1:decimal(38,10)>", readerInspector.getTypeName());
```

Comment:
Change in decimal scale. Expected?

Reply:
Seems that that type scale was not properly propaged before and was using HiveDecimal.SYSTEM_DEFAULT_SCALE which is 18: https://github.com/apache/hive/blob/ff6f3565e50148b7bcfbcf19b970379f2bd59290/ql/src/java/org/apache/hadoop/hive/ql/io/orc/OrcStruct.java#L607

This is actually in-line with ORC tests for the same file:
https://github.com/apache/orc/blob/b54d10cedf5ec1529cf06d77268510c216402cba/java/core/src/test/org/apache/orc/impl/TestRecordReaderImpl.java#L141